content/en/security/application_security/how-appsec-works.md

```diff
@@ -89,8 +87,7 @@ To configure the information redacted by ASM, refer to the [data security config
 
 Datadog uses multiple pattern sources, including the [OWASP ModSecurity Core Rule Set][12] to detect known threats and vulnerabilities in HTTP requests. When an HTTP request matches one of [the OOTB detection rules][13], a security signal is generated in Datadog.
 
-<div class="alert alert-info"><strong>Beta: Automatic Threat Patterns Updates</strong><br>
-If your service is running with <a href="/agent/guide/how_remote_config_works/#enabling-remote-configuration">an Agent with Remote Configuration enabled and a tracing library version that supports it</a>, the threat patterns being used to monitor your service is automatically updated whenever Datadog publishes updates.</div>
+**Automatic Threat Patterns Updates:** If your service is running with <a href="/agent/guide/how_remote_config_works/#enabling-remote-configuration">an Agent with Remote Configuration enabled and a tracing library version that supports it</a>, the threat patterns being used to monitor your service is automatically updated whenever Datadog publishes updates.
```

Comment:
```suggestion
**Automatic Threat Patterns Updates:** If your service is running with <a href="/agent/guide/how_remote_config_works/#enabling-remote-configuration">an Agent with Remote Configuration enabled and a tracing library version that supports it</a>, the threat patterns used to monitor your service are automatically updated whenever Datadog publishes updates.
```

Reply:
Since it's not in a `<div></div>` anymore, this link can be updated to a Markdown-style reference link